src/week1/ElectricAppliancesAutomation.java

```diff
@@ -0,0 +1,5 @@
+package week1;
+
+public interface ElectricAppliancesAutomation {
+    public void automateElectricAppliances(boolean sensorMsg);
```

Comment:
method name should be automate. Create Top interface Automation which would then be extended by ElectricAppiance and WaterMotor

Reply:
Hemant, here method signature is different for both the methods I can't change their name or can't create top level interface for this